src/mapper.rs

```diff
@@ -61,13 +61,16 @@ struct MemberMapping<'s> {
     original_startline: usize,
     original_endline: Option<usize>,
     is_synthesized: bool,
+    is_outline: bool,
```

Comment:
All members are always going to have the same value for this field as the method as a whole. Therefore I would pick one place to put it, either here or in `ClassMembers`. `ClassMembers` has the advantage that we only store it once, `MemberMapping` has the advantage that `is_synthesized` lives there and the two are treated the same.

(Morally `is_synthesized` probably also shouldn't be on `MemberMapping` because every member is going to have the same value, it was just more convenient to build it that way.)